app/views/councillor_contributions/_contribution_form_input.html.haml

```diff
@@ -0,0 +1,15 @@
+%fieldset
+  %legend
```

Comment:
Is the indentation here wrong? It appears to me like the form fields will be indented inside the `<legend>`, which they should not be.